RELEASE_NOTES.md

```diff
@@ -1,5 +1,10 @@
 # Change log
 
+## in develop
+
+* Updates build environment to SBT 1.6.1
```

Comment:
I don't think we need release notes for these changes that are only developer-facing. @YuxinShi0423 @jdidion

Reply:
I will remove the scalafmt line but for sbt we have similar notes below so I will keep it for now.